eox_tenant/apps.py

```diff
@@ -51,3 +51,9 @@ def ready(self):
         """
         from eox_tenant.permissions import load_permissions
         load_permissions()
+
+        if base_settings.EOX_TENANT_CHANGE_ENROLLMENT_MANAGER:
+            from eox_tenant.edxapp_wrapper.enrollments import get_enrollments
+            from eox_tenant.models import EdnxCourseEnrollmentManager
+            enrollments = get_enrollments()
+            enrollments.objects = EdnxCourseEnrollmentManager()
```

Comment:
does it work as we expect?

Reply:
I mean, this is doing the filtering as we expect? If the answer is yes, that's supercool

Reply:
It's not working :( I had to do this `enrollments._meta.default_manager_name = EdnxCourseEnrollmentManager()` to make the code to run at least but the results are not being filtered as expected and also it's not a good practice.